bundle/server.go

```diff
@@ -51,6 +39,27 @@ func (s *Server) HostBundle(name string, files map[string]string) error {
 	return nil
 }
 
+func HostBundleWithDataYaml(name string, files map[string]string, dataYaml string) error {
+	files["data.yaml"] = dataYaml
```

Comment:
note the way our Rego works currently is Running all rules if the benchmark isn't listed. we should remember that on the integration side when "building" the data.yaml